Header header

int32 wp_type
float64[] x
float64[] y
float64[] yaw
float64[] v
float64[] curv